docs/explainers/cli-vs-api.md

```diff
@@ -0,0 +1,41 @@
+# Cloud Foundry CLI vs API
+
+## What are the Cloud Foundry CLI and API?
+
+Cloud Foundry provides users several ways to interact with resources outside of applications such as this one: the command line (CLI) and API. The CLI's behavior may be different from similar functionality in the API, and even different versions of the CLI and API may have variations in behavior.
+
+### The versions you may come across
+
+- API
+  - /v2 -- deprecated, still in use by older software such as Stratos
+  - /v3 -- same database as /v2 but very different patterns of access
```

Comment:
[blocking request] We should explain somewhere that our app is trying to use API v3 whenever possible, with V2 as a fallback.

Reply:
Good thinking, will do!